### PR TITLE
triton vscode: updates to improve it as a landing page

### PR DESCRIPTION
**Pull request by rkdarst**
- Improve info on Remote SSH and the warnings of using it
- Link to this page from a few more places
- General reorganizations
- Review: worth a detailed read and thinking what else needs to be said.

### CODE REVIEW

File: triton/apps/vscode.rst [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+
+
+VSCode remote SSH
+-----------------
+
+**"Remote SSH"** is a nice way to work on a remote computer and
+provides both editing and shell access, but everything will run
+directly on the login node on Triton.  This is OK for editing, but not
+for main computations (see the section above or below).  **To repeat:
+don't use this for running big computations.**
```
Comment (bast) [inline triton/apps/vscode.rst:33]: Sometimes I get the very good question: How can I find out that I am running a computation on login node?

**Comment by bast** [comment]
Some screenshots can be useful like here: https://documentation.sigma2.no/code_development/guides/vs_code/connect_to_server.html

File: triton/apps/vscode.rst (hunk 1 above)
Comment (rkdarst) [reply triton/apps/vscode.rst:33]: Added a screenshot mentioning this (but I'm not entirely sure it will show the same for everyone!)

**Comment by rkdarst** [comment]
> Some screenshots can be useful like here: https://documentation.sigma2.no/code_development/guides/vs_code/connect_to_server.html

I linked to this.

**Comment by bast** [comment]
Thanks! This is great documentation.

**Comment by eglerean** [comment]
Merging and closing. We can revise if things have changed since last July.

**Pull request merged**